pkg/cmd/kafka/acl/admin/admin.go

```diff
@@ -108,6 +110,7 @@ func NewAdminACLCommand(f *factory.Factory) *cobra.Command {
 	fs.AddUser(&userID)
 	fs.AddServiceAccount(&serviceAccount)
 	fs.AddAllAccounts(&allAccounts)
+	fs.AddInstanceID(&opts.kafkaID)
```

Comment:
Oh. wow. If this is soo easy now we can add that to every Kafka topic/consumer group command